Cargo.toml

```diff
@@ -146,6 +148,8 @@ serde_cbor = "0.11.2"
 serde_json = "1.0.114"
 serde-pickle = "1.1.1"
 serde_yaml = "0.9.19"
+# todo: stabby will release crate version with build fix soon
+stabby = { git = "https://github.com/ZettaScaleLabs/stabby.git" }
```

Comment:
@p-avital any estimate on stabby? As you know nothing can be released on `crates.io` while pointing to some `git` dependency :)

Reply:
@Mallets I can see that the needed fix is released in 4.0.5, so I will switch to it